src/Microsoft.Extensions.Logging.Abstractions/ILogger.cs

```diff
@@ -8,24 +8,24 @@ namespace Microsoft.Extensions.Logging
     /// <summary>
     /// Represents a type used to perform logging.
     /// </summary>
+    /// <remarks>Aggregates most logging patterns to a single method.</remarks>
     public interface ILogger
     {
         /// <summary>
-        /// Aggregates most logging patterns to a single method.
+        /// Writes an event to the logger.
```

Comment:
I like the 'entry' also better, but there is a parameter 'eventId' ? (and not entryId)